sqlglot/optimizer/qualify.py

```diff
@@ -28,6 +29,7 @@ def qualify(
     validate_qualify_columns: bool = True,
     quote_identifiers: bool = True,
     identify: bool = True,
+    pushdown_cte_alias_columns: t.Optional[bool] = None,
```

Comment:
this option is unnecessary as it comes from the dialect

Reply:
Makes sense. I removed it - had it initially before I added it to the dialect.